eve/workers/end2end/scripts/run-e2e-test.sh

```diff
@@ -17,6 +17,13 @@ POD_NAME="${ZENKO_NAME}-${STAGE}-test"
 TOKEN=$(get_token)
 
 # set environment vars
+MONGO_DATABASE=$(kubectl get secret end2end-connector-cloudserver-config.v4 -o jsonpath="{.data.config\.json}}" | base64 -di  | jq -r '.mongodb.database')
```

Comment:
suggestion to reduce the number of repeated text:

```suggestion
CLOUDSERVER_SECRET_V4=$(kubectl get secret end2end-connector-cloudserver-config.v4 -o jsonpath="{.data.config\.json}}" | base64 -di)

MONGO_DATABASE=$($CLOUDSERVER_SECRET_V4 | jq -r '.mongodb.database')
...
```